src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,68 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+import { getMovie, normalizeMovieData } from '../../api';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  addMovie: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [movieData, setMovieData] = useState<MovieData | null>(null);
+
+  const handleSubmit = useCallback(
+    (e: React.FormEvent<HTMLFormElement>) => {
+      e.preventDefault();
+      setIsLoading(true);
+
+      getMovie(query.trim())
+        .then((data) => {
+          if ('Error' in data) {
```

Comment:
Error will be in catch
Remove redundant checker

Reply:
the error is handle at the top level, so I am checking that we received MovieData or ResponseError